site/content/contribute/webapp/end-to-end-tests.md

```diff
@@ -25,6 +25,27 @@ Not familiar with Cypress? Here is some documentation that will help you get sta
     - Initiate via node CLI to selectively run specs based on test metadata, e.g. `node run_tests.js --group='@accessibility'` which will run all specs with `@accessibility` metadata.
 3.  Tests are executed according to your selection and will display whether the tests passed or failed.
 
+### Environment Variables
+There are several environment variables we're using for Cypress testing with the primary purpose:
+- to easily change when running in CI, and
+- to cater different values across developer machines.
+
+Environment variables are defined in "[cypress.json](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress.json)" under `env` key. In most cases, you don't need to change the values as it make use of the default local developer setup, but when you do, the easiest is to override by exporting `CYPRESS_*`, where `*` is the key of the variable like `CYPRESS_adminUsername`. See [Cypress documentation](https://docs.cypress.io/guides/guides/environment-variables.html#Setting) for details.
```

Comment:
```suggestion
Environment variables are [defined in `cypress.json`](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress.json)" under the `env` key. In most cases, you don't need to change the values as it makes use of the default local developer setup. However, when you do the easiest method is to override by exporting `CYPRESS_*`, where `*` is the key of the variable, for example: `CYPRESS_adminUsername`. See [Cypress documentation](https://docs.cypress.io/guides/guides/environment-variables.html#Setting) for details.
```

Reply:
I'm not 100% sure that the suggested formatting for cypress.json will render properly, if it doesn't it can be discarded.

Reply:
Backticks don't have any effect, and so I removed it.

Done.